regression/iterate_data.py

```diff
@@ -75,12 +54,14 @@ def get_file_count(cloud_url: str) -> int:
 
         bucket = Bucket(gcs_client, obj.netloc)
         files = bucket.list_blobs(prefix=obj.path.lstrip('/'))
+        return sum(1 for f in files if f.key[-1] != '/')
```

Comment:
What is this doing?

Reply:
looks like the variable `files` contains the directory as well as files and when counting, it accounts for directory as well which is incorrect. Hence the `f.key[-1] != '/'` ignores the directory and only counts the file.